frame/lending/src/tests.rs

```diff
@@ -1589,9 +1586,12 @@ fn market_owner_cannot_retroactively_liquidate() {
 			max_price_age: DEFAULT_MAX_PRICE_AGE,
 		};
 		// ALICE is the creater of the market.
-		assert_ok!(Lending::update_market(Origin::signed(*ALICE), market_id, updatable));
-		// BOB loan must be liquidated now.
-		assert_eq!(Lending::should_liquidate(&market_id, &BOB), Ok(true));
+		assert_noop!(
+			Lending::update_market(Origin::signed(*ALICE), market_id, updatable),
+			Error::<Runtime>::CannotIncreaseCollateralFactorOfOpenMarket
+		);
+		// if above update was succeeded BOB's loan would have to be liquidated.
+		assert_eq!(Lending::should_liquidate(&market_id, &BOB), Ok(false));
```

Comment:
Should we drop this assert?

Reply:
@mikolaichuk ensure macro retruns error if its condition is not satisfied . So my original check was correct and your check is wrong
```rust
					ensure!(
						input.collateral_factor > market.collateral_factor,
						Error::<T>::CannotIncreaseCollateralFactorOfOpenMarket
					);
```
with above check function will only allow updates when input > market's collateral.

Reply:
Also after my change updates which increases collateral factor should not be allowed and hence following assert is correct.
```rust
assert_eq!(Lending::should_liquidate(&market_id, &BOB), Ok(false));
```